examples/action_dann_lightn/model.py

```diff
@@ -64,7 +65,8 @@ def get_feat_extractor(model_name, num_classes, num_channels):
     """
 
     if model_name == 'I3D':
-        feature_network = i3d(name='rgb_imagenet', pretrained=True)
+        pt_name = 'rgb_imagenet' if num_channels == 3 else 'flow_imagenet'
```

Comment:
pt can be made more readable. I can not figure out what it stands for. Part (if yes, part_name is better)?